src/responses.rs

```diff
@@ -19,21 +19,20 @@ pub struct OutpackError {
     pub detail: String,
 
     #[serde(skip_serializing, skip_deserializing)]
-    pub kind: Option<ErrorKind>
+    pub kind: Option<ErrorKind>,
 }
 
 impl OutpackError {
-    pub fn new(e: io::Error) -> OutpackError {
+    pub fn from(e: io::Error) -> OutpackError {
```

Comment:
This is unrelated but `from` seems more correct and we might reserve `new` for directly creating an error object by passing explicit `kind`, `error` and `detail` constructor arguments.